trieTools/updateTokenIdChecker/main.go

```diff
@@ -0,0 +1,221 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"os"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-go/common"
+	"github.com/multiversx/mx-chain-go/common/errChan"
+	"github.com/multiversx/mx-chain-go/state"
+	"github.com/multiversx/mx-chain-go/vm/systemSmartContracts"
+	logger "github.com/multiversx/mx-chain-logger-go"
+	"github.com/multiversx/mx-chain-tools-go/trieTools/trieToolsCommon"
+	"github.com/urfave/cli"
+)
+
+var log = logger.GetOrCreate("trie")
+
+const (
+	logFilePrefix  = "trie"
+	rootHashLength = 32
+	addressLength  = 32
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "Trie stats CLI app"
+	app.Usage = "This is the entry point for the tool that prints stats about the state"
+	app.Flags = getFlags()
+	app.Authors = []cli.Author{
+		{
+			Name:  "The MultiversX Team",
+			Email: "contact@multiversx.com",
+		},
+	}
+
+	app.Action = func(c *cli.Context) error {
+		return startProcess(c)
+	}
+
+	err := app.Run(os.Args)
+	if err != nil {
+		log.Error(err.Error())
+		os.Exit(1)
+		return
+	}
+
+	log.Info("execution finished successfully")
+}
+
+func startProcess(c *cli.Context) error {
+	flagsConfig := getFlagsConfig(c)
+
+	_, errLogger := trieToolsCommon.AttachFileLogger(log, logFilePrefix, flagsConfig.ContextFlagsConfig)
+	if errLogger != nil {
+		return errLogger
+	}
+
+	log.Info("sanity checks...")
+
+	err := logger.SetLogLevel(flagsConfig.LogLevel)
+	if err != nil {
+		return err
+	}
+
+	shardsState, err := LoadStateForAllShards(flagsConfig)
+	if err != nil {
+		return err
+	}
+
+	systemEsdtAddress := "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u"
+	systemEsdtAccount, err := GetAccountFromBech32String(systemEsdtAddress, shardsState[Meta])
+	if err != nil {
+		return err
+	}
+
+	tokensMap, err := getAllESDTsFromSystemEsdtAccount(systemEsdtAccount)
+	if err != nil {
+		return err
+	}
+
+	systemAccounts, err := getSystemAccountsForShards(shardsState)
+	if err != nil {
+		return err
+	}
+
+	checkUpdateTokenTypeCalled(systemAccounts, tokensMap)
+	return nil
+}
+
+func checkUpdateTokenTypeCalled(systemAccounts map[ShardID]state.UserAccountHandler, tokensMap map[string][][]byte) {
+	ESDTPrefix := []byte("ELRONDesdt")
```

Comment:
local variable with upper case letters should change to lower case

Reply:
Done.